src/components/CleanHtml.tsx

```diff
@@ -0,0 +1,8 @@
+import dompurify from 'dompurify';
+
+const CleanHtml = ({ dirtyHtml }: { dirtyHtml: string }): JSX.Element => {
```

Comment:
Another useful property might be to allow the user of the component to select what HTML element to use, i.e. add a `component` property that is a string that defaults to "div".

Reply:
you could even type the `component` property with the same type the `Box` component uses, then it will only allow valid html tags